src/parser/monk/mistweaver/modules/spells/EnvelopingMists.js

```diff
@@ -24,15 +23,32 @@ class EnvelopingMists extends Analyzer {
     combatants: Combatants,
   };
 
+  constructor(...args) {
+    super(...args);
+    if (this.selectedCombatant.hasTalent(SPELLS.MIST_WRAP_TALENT.id)) {
+      this.EVM_HEALING_INCREASE=.4;
+    }
+  }
+
+  EVM_HEALING_INCREASE = 0.3;//check for mistwrap
   healingIncrease = 0;
   gustsHealing = 0;
   lastCastTarget = null;
+  countForGusts = false;
```

Comment:
Is this variable needed somewhere else?

Reply:
This is an artifact of other attempts to fix the mastery issue